recipes/scotch/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "scotch" %}
+{% set version = "6.0.4" %}
+{% set md5 = "d58b825eb95e1db77efe8c6ff42d329f" %}
+{% set file_url = "http://gforge.inria.fr/frs/download.php/file/34618/scotch_6.0.4.tar.gz" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: scotch_{{ version }}.tar.gz
+  md5: {{ md5 }}
+  url: {{ file_url }}
+  patches:
+    - clock_gettime.patch  # [osx]
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+ build:
+  - gcc
```

Comment:
I reversed the change again, because it didn't compile:

```
/opt/rh/devtoolset-2/root/usr/libexec/gcc/x86_64-redhat-linux/4.8.2/ld: cannot find -lz
```